petab/core.py

```diff
@@ -90,7 +90,7 @@ def get_observable_id(parameter_id: str) -> str:
 
 
 def flatten_timepoint_specific_output_overrides(
-        petab_problem: problem.Problem) -> None:
+        petab_problem: 'problem.Problem') -> None:
```

Comment:
this is to avoid circular imports?

Reply:
yes, pytest had troubles for reasons i didn't fully understand